### ISSUE TITLE
Add functionality to sort the environment

### ISSUE THREAD

**Issue by Ayutac** [digama0/mizar-rs#7]
The original Mizar implementation is missing a way to sort the environment. There is an unofficial perl script flying around for notations, though. It would be nice to have a sorting command in mizar-rs.

**Requirements**

- [ ] a command to order the environment in the order given by mml.lar
- [ ] a flag to exclude notations, since notations are sometimes in another order on purpose

**Comment by Ayutac** [comment]
I have done the core implementation of this in my experimental branch, but what I am missing is a way to rewrite the article. Is there functionality for that already in mizar-rs?

**Comment by digama0** [comment]
No, currently there are no tools for writing or editing .miz files, so that would have to be a component or prerequisite for this issue.
